util/pbtypes.ts

```diff
@@ -0,0 +1,50 @@
+/**
```

Comment:
should we include this file or should this be .gitignored?

Reply:
since its generated

Reply:
it's generated but it shouldn't get gitignored because other devs normally don't have access to the database

Reply:
there's also no sensitive info in the file